_posts/devops/2016-04-17-docker-getting-started.markdown

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: Docker - Getting Started
+author: Harshad_Yeola
+modified:
+comments: true
+categories: devops
+excerpt:
+tags: [devops]
```

Comment:
@MiteshShah do you mean `_posts/devops/docker/2016-04-17-docker-getting-started.markdown`?